docs/integrations/builtin/credentials/wise.md

```diff
@@ -38,3 +38,29 @@ The following video demonstrates the steps mentioned above.
 <iframe width="840" height="472.5" src="https://www.youtube.com/embed/hys2lDEScUE" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 </div>
 
+
+## Personal Token SCA
+
+When making a request to an SCA protected endpoint, a 403 Forbidden HTTP status code is returned.
+SCA is required on Get Statements, and Funds transfer request. It is disabled when using test endpoints.
```

Comment:
Heya! Not 100% clear what is being referred to here. If possible, could you match to the operation names in n8n? So for example, if SCA is required for Transfer > Create operations, or for all Transfer operations?

Reply:
Its not clear where SCA is needed. So and to my knowledge  it is `Get Statements` and `Funds`

Reply:
```suggestion
SCA is required on some operations. Refer to [Wise documentation | Strong Customer Authentication & 2FA](https://docs.wise.com/api-docs/features/strong-customer-authentication-2fa) for details. It is disabled when using test endpoints.
```